research/gaia/gremlin/gremlin_core/tests/common/mod.rs

```diff
@@ -216,12 +216,12 @@ pub mod test {
                             {
                                 let mut tag_entries = vec![];
                                 for (tag, one_tag_value) in result_prop.tag_entries.iter() {
-                                    let tag_entry: Vec<(String, Object)> = if let Some(element) =
+                                    let tag_entry: Vec<(PropKey, Object)> = if let Some(element) =
                                         one_tag_value.graph_element.as_ref()
                                     {
-                                        vec![("".to_string(), element.id().into())]
+                                        vec![("".to_string().into(), element.id().into())]
```

Comment:
Suggest implement From(&str) for PropKey

Reply:
Done